Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -87,17 +85,17 @@ public virtual void Initialize(IAlgorithm algorithm, IBrokerage brokerage, IResu
             _brokerage = brokerage;
             _brokerage.OrderStatusChanged += (sender, fill) =>
             {
-                HandleOrderEvent(fill);
+                Process(fill);
             };
 
             _brokerage.SecurityHoldingUpdated += (sender, holding) =>
             {
-                HandleSecurityHoldingUpdated(holding);
+                Process(holding);
             };
 
             _brokerage.AccountChanged += (sender, account) =>
             {
-                HandleAccountChanged(account);
+                Process(account);
```

Comment:
Please inline these `Process` method implementations as all they're doing is calling `Enqueue` on the appropriate event queue.